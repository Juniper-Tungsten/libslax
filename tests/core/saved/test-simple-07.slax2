/* Machine Crafted with Care (tm) by slaxWriter */
version 1.1;

match * {
    mode "one";
    
    <one> .;
}

match * {
    mode "two";
    
    <two> string-length(.);
}

match random {
    mode "two";
    
    <xxx> string-length(.);
}

match doc {
    <top> {
        apply-templates version {
            mode "one";
        }
        apply-templates version {
            mode "two";
        }
    }
}
