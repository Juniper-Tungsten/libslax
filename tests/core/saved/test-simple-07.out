<?xml version="1.0"?>
<top>
  <one>7.1R1.4</one>
  <two>7</two>
</top>
